shenyu-plugin/shenyu-plugin-tars/src/main/java/org/apache/shenyu/plugin/tars/util/PrxInfoUtil.java

```diff
@@ -141,9 +141,11 @@ public static String getObjectName(final String upstreamUrl, final String servic
     public static Object[] getParamArray(final Class<?>[] paramTypes, final String[] paramNames, final String body) {
         Map<String, Object> bodyMap = GsonUtils.getInstance().convertToMap(body);
         Object[] param = new Object[paramNames.length];
+        String paramName;
+        Class<?> paramType;
         for (int i = 0; i < paramNames.length; i++) {
-            String paramName = paramNames[i];
-            Class<?> paramType = paramTypes[i];
+            paramName = paramNames[i];
+            paramType = paramTypes[i];
```

Comment:
I dont think change like this is a good choise.

Reply:
I will modify and resubmit.